skate-plugin/project-gen/src/jvmMain/kotlin/slack/tooling/projectgen/circuitgen/CircuitGenClassName.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2024 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package slack.tooling.projectgen.circuitgen
+
+import com.squareup.kotlinpoet.ClassName
+
+class CircuitGenClassNames {
+  companion object {
```

Comment:
You can just make `CircuitGenClassNames` an object

```suggestion
object CircuitGenClassNames {
```

Reply:
https://github.com/slackhq/slack-gradle-plugin/pull/799/commits/70b29708ba841e1850dd82a7995675fa598ae83b